### PR TITLE
Consolidate community and enterprise edition builds

### PR DESCRIPTION
**Pull request by tobim**

### CODE REVIEW

**Review by mavam — APPROVED** [review]

File: .github/workflows/docker-manual.yaml [hunk 1]
```diff
@@ -89,15 +85,6 @@ jobs:
                 }
               else
                 empty
-              end,
-              if ${{ inputs.push-ee-to-ecr }} then
-                {
-                  "name": "tenzir-ee", "target": "tenzir-ee", "registries": [
-                   "622024652768.dkr.ecr.eu-west-1.amazonaws.com"
```
Comment (dominiklohmann) [inline .github/workflows/docker-manual.yaml:96]: @tobim doesn't this break things? We don't push the other image to ECR.

Comment (tobim) [reply .github/workflows/docker-manual.yaml:96]: This file is only used for the manual dispatched workflow. The only image that is automatically pushed to ECR is `tenzir-demo`, and that is unchanged by this PR.

File: nix/overlay.nix [hunk 2]
```diff
@@ -384,24 +384,14 @@ in {
...
-        pname = "tenzir-ee";
-      };
     in
       pkg.withPlugins (ps: [
         ps.compaction
         ps.context
         ps.matcher
         ps.pipeline-manager
         ps.platform
+        ps.vast
```
Comment (dominiklohmann) [inline nix/overlay.nix:394]: Why do we need to define all the plugins here again when they're already defined here? https://github.com/tenzir/tenzir/blob/topic/unify-builds/nix/tenzir/plugins/names.nix

Comment (tobim) [reply nix/overlay.nix:394]: Now that we have the complete list here we can do that, but we couldn't before when we had individual edition builds. This code could be refactored to reflect that change, but I didn't have the time to do it yet.

Comment (dominiklohmann) [reply nix/overlay.nix:394]: Okay, I'll write a story for this.

**Pull request merged**